includes/ApplHandlingByStr.php

```diff
@@ -82,6 +86,22 @@ function showFullStatus( $stat_str )
 }
 
 
+function findPending( $stat_str )
+{
+	$STR = explode ( ';', $stat_str );
+	for ( $i =  1;  $i < count ( $STR ) ; $i++ )
+	{
+		$str = explode ( ',', $STR[$i] );
+		$usr = $str[0];
+		
+		if( $str[1] == 'P' )
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed
* Expected 1 space after IF keyword; 0 found
* Expected 0 spaces after opening bracket; 1 found
* Expected 0 spaces before closing bracket; 1 found
* Expected 1 space after closing parenthesis; found 9

<!-- Comment by [SideCI](https://sideci.com) -->